py3status/modules/net_iplist.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+"""
+Display the list of current IPs.
+
+This can exclude specified IPs and displays "no connection" if there are no IPs
+to display.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds.
+        (default 30)
+    format: format of the output.
+        (default 'Network: {format_iface}')
+    format_iface: format string for the list of IPs of each interface.
+        (default '{iface}: v4{{{ip}}} v6{{{ip6}}}')
+    format_no_ip: string to show if there are no IPs to display.
+        (default 'no connection')
+    iface_blacklist: list of interfaces to ignore. Accepts shell-style wildcards.
+        (default ['lo'])
+    iface_sep: string to write between interfaces.
+        (default ' ')
+    ip_blacklist: list of IPs to ignore. Accepts shell-style wildcards.
+        (default [])
+    ip_sep: string to write between IP addresses.
+        (default ',')
+    remove_empty: do not show interfaces with no IP.
+        (default True)
+
+Format placeholders:
+    {format_iface} the format_iface string.
+
+Format placeholders for format_iface:
+    {iface} name of the interface.
+    {ip} list of IPv4 of the interface.
+    {ip6} list of IPv6 of the interface.
+
+Color options:
+    color_bad: no IPs to show
+    color_good: IPs to show
+
+Example:
+
+```
+net_iplist {
+    iface_blacklist = []
+    ip_blacklist = ['127.*', '::1']
+}
+```
+
+Requires:
+    ip: utility found in iproute2 package
+"""
+
+
+import re
+from subprocess import check_output
+from fnmatch import fnmatch
+
+
+class Py3status:
+    cache_timeout = 30
+    format = 'Network: {format_iface}'
+    format_iface = '{iface}: v4{{{ip}}} v6{{{ip6}}}'
```

Comment:
how about `'{iface}:[ {ip}][ {ip6}]'`

One of my interfaces has no IPv6 and it looks funny being blank.  Also I find the `{` an unnecessary distraction.